serializer/stream.go

```diff
@@ -0,0 +1,9 @@
+package serializer
+
+import "io"
+
+// Stream low level Reader, Writer implementation of a underlaing data stream.
+type Stream struct {
```

Comment:
Why is it beneficial for users to have a struct that's both a Reader and Writer? The deserializing methods are implemented in the vm package already, this seems like it could be confusing if you're just trying to read the contents of a file (or write some stuff to a file).

Reply:
I currently created the stream struct for testing purposes. This could later on be changed and split into a Read and Writer interface/struct.